python/app/ui/resources_rc.py

```diff
@@ -6,7 +6,10 @@
 #
 # WARNING! All changes made in this file will be lost!
 
-from tank.platform.qt import QtCore
+try:
```

Comment:
We can't do this. As noted above, the contents of this file will change as resources are added/modified/removed. Perhaps we should have a separate resource build for flame, but that doesn't seem right.I can think of some really hacky ways to get around this problem, but I don't want to recommend them. 😐